src/index.js

```diff
@@ -570,7 +573,7 @@ export function app(props) {
       if (typeof obj[0] === "function") {
         dispatch(obj[0](state, obj[1], data))
       } else {
-        obj[1].effect(obj[1], dispatch, setState(obj[0]))
+        obj[1][0](obj[1], dispatch, setState(obj[0]))
```

Comment:
Shouldn't this be the following?
```js
obj[1][0](obj[1][1], dispatch, setState(obj[0]))
```

Reply:
I can get behind that. I was hesitate to make the change since previously the first param was the whole effect object.

Reply:
With that in mind, I'll update subscriptions to behave the same way.

Reply:
I think when the whole effect is an object it makes more sense than having an array that users will always need to be doing `props[1]` with.

Reply:
Unless there's a use case I'm not thinking of for the effect to need a reference to its own function while running for some reason 🤔

Reply:
I can't think of a good reason, since I should know which function is being called at this point.